crates/apub/src/activities/mod.rs

```diff
@@ -136,6 +141,24 @@ pub(crate) fn verify_visibility(
   }
 }
 
+pub(crate) async fn update_community_tags(
+  context: &Data<LemmyContext>,
+  community_id: CommunityId,
+  group_url: String,
+  post_tags: Vec<LemmyCommunityTag>,
+) -> LemmyResult<()> {
+  Tag::community_override_all_from_apub(
+    &mut context.pool(),
+    community_id,
+    group_url,
+    post_tags
+      .into_iter()
+      .map(|t| t.into_tag_insert_form(community_id))
+      .collect(),
+  )
+  .await
+}
```

Comment:
This function seems pointless, you can convert to tag inside `community_override_all_from_apub`

Reply:
I can not because the DB schema crate does not depend on the API crate